### ISSUE TITLE
Autoplay youtube

### ISSUE THREAD

**Issue by viliusle** [rqrauhvmra/Tobii#69]
Videos in pages usually should not auto-play, its annoying.
But when you click on video to get modal dialog, it should always auto-play when modal dialog loads, because if not, user have to click 2 times to start watching video.

p.s. `autoplayVideo` should not exists or have true value by default.

**Comment by midzer** [comment]
I wished this parameter ages ago for those `<video>` in https://feuerwehr-eisolzried.de/media/

We can remove this in a v3 (breaking change) and set its behaviour to true, yes.

**Comment by midzer** [comment]
All media autoplay now in v3 https://github.com/midzer/tobii/commit/da84e29194fd520f5eb835cc7a3fad49581d71c1

**Comment by smcyr** [comment]
I think that the option should still be there. It's not good accessibility if you have a slider and one of the elements is a video and suddenly, when you swipe, the video start to play automatically. It would be really appreciated to have the option back.

**Comment by viliusle** [comment]
True, this is rare case where you may want to turn auto play off. In such case I have few thoughts:

1. what about auto play video by default if it was first opened element, and if user clicks next to load another video, detect that and automatically disable auto play so user can do next, next next, pick one video and play it. So first video plays automatically, but not any other videos.
2. I personally dont like gallery of videos in modal dialogs. In most cases it is not user friendly. Like you click next to skip 10 seconds and ... next video loads :) Maybe such situation should not exist and all videos should not be grouped? But that could be only my opinion.

**Comment by smcyr** [comment]
Both points are good points, but while I would agree for point 2, it can be dependant on what a client want. In any case, the first option would be good at least, to disable the autoplay when you slide to a video.

**Comment by midzer** [comment]
PR welcome in https://github.com/midzer/tobii/tree/develop :)
